src/FamilyHubs.ReferralUi.Ui/Pages/ProfessionalReferral/ContactDetails.cshtml.cs

```diff
@@ -9,68 +11,61 @@ namespace FamilyHubs.ReferralUi.Ui.Pages.ProfessionalReferral;
 [Authorize(Policy = "Referrer")]
 public partial class ContactDetailsModel : PageModel
 {
-    [BindProperty]
-    public string ReferralId { get; set; } = default!;
-    [BindProperty]
-    public string FullName { get; set; } = default!;
+    private readonly IRedisCacheService _redisCacheService;
 
     [BindProperty]
     public List<string> ContactSelection { get; set; } = new List<string>();
 
-
     [BindProperty]
     [EmailAddress(ErrorMessage = "Please enter a valid email address")]
-    public string? Email { get; set; } = default!;
+    public string? Email { get; set; }
 
     [BindProperty]
     public bool EmailValid { get; set; } = true;
 
     [BindProperty]
     [Phone(ErrorMessage = "Please enter a valid phone number")]
-    public string? Telephone { get; set; } = default!;
+    public string? Telephone { get; set; }
 
     [BindProperty]
     public bool TelephoneValid { get; set; } = true;
 
+    public string FullName { get; set; } = default!;
```

Comment:
The other strings in the model are nullable.
Does FullName need to be non-nullable and set to default?
Setting a non-nullable to default should only really be used when necessary, such as with types that are used for deserialisation.

Reply:
Replaced FullName with string.Empty, Email, Telephone, Textphone are option, although you must pick one.